src/modules/movieCount.test.js

```diff
@@ -0,0 +1,35 @@
+// const fetch = require('node-fetch');
```

Comment:
- [ ] When I run your tests, I get an error on my terminal for movie count as seen below 
![image](https://github.com/rbd3/JavascriptCapstone/assets/45460192/cb3f6d8f-3ca7-4ca0-be0d-7b31427dd890)

- [ ] I also noticed that you only covered a few test cases such as the moviecount and total number of comments. Please cover more test cases such as the number of likes and so on(As one of the requirements, all counters should be covered with unit tests). You should also include scenarios where there are zero items on the page as one of your test cases